x-pack/plugins/enterprise_search/public/applications/app_search/components/crawler/types.ts

```diff
@@ -199,36 +199,39 @@ export interface CrawlRequest {
   completedAt: string | null;
 }
 
-export type CrawlEventStage = 'crawl' | 'process';
-
 export interface CrawlConfig {
   domainAllowlist: string[];
 }
 
 export interface CrawlConfigFromServer {
   domain_allowlist: string[];
 }
-export interface CrawlEventFromServer {
-  id: string;
+
+export type CrawlRequestWithDetailsFromServer = CrawlRequestFromServer & {
+  type: CrawlType;
+  crawl_config: CrawlConfigFromServer;
+  // TODO add other properties like stats
```

Comment:
I assume you will add this in another PR when we are ready to show the stats in the flyout?

Reply:
Yep, wanted to leave a note for myself or if someone else picked this up